### DISCUSSION TITLE
Permission denied error on python plugin

### DISCUSSION THREAD

**Discussion by lohxx** [Kong/kong#9805] [Help] [answered]
I am having the error: `Permission denied`, when I configure my custom python plugin. Can anyone help me? 

The plugin configuration is: 
```
      KONG_PLUGINS: bundled,authplugin
      KONG_PLUGINSERVER_NAMES: authplugin
      KONG_PLUGINSERVER_AUTHPLUGIN_SOCKET: /auth-plugin.socket
      KONG_PLUGINSERVER_AUTHPLUGIN_START_CMD: /auth-plugin/auth.py
      KONG_PLUGINSERVER_AUTHPLUGIN_QUERY_CMD: /auth-plugin/auth.py --dump
```
The plugin is on `/` directory.

![image](https://user-images.githubusercontent.com/26140123/203385765-e947cc8c-f64e-42be-b5c9-690ad6b776fb.png)

**Comment by lohxx** [comment, marked as answer]
I fixed the problem making the file executable with chmod.

I added this line to dockerfile:
```dockerfile
RUN chmod +x /auth-plugin/auth.py
```

**Reply by fffonion** [reply]
correct you need the executable permission, or adding the interpreter (assume python3)
`/usr/bin/python3 /auth-plugin/auth.py`
